python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -5847,8 +5880,45 @@ tables:
   - name: diaSourceId
     description: Unique identifier of this DiaSource.
     datatype: long
+  constraints:
+  - name: FK_DiaSource_Visit
+    description: Link a DiaSource to its associated Visit.
+    '@type': ForeignKey
+    columns:
+    - '#DiaSource.visit'
+    referencedColumns:
+    - '#Visit.visit'
+  - name: FK_DiaSource_DiaObject
+    description: Link a DiaSource to its associated DiaObject.
+    '@type': ForeignKey
+    columns:
+    - '#DiaSource.diaObjectId'
+    referencedColumns:
+    - '#DiaObject.diaObjectId'
+  - name: FK_DiaSource_SSObject
+    description: Link a DiaSource to its associated SSObject, if any.
+    '@type': ForeignKey
+    columns:
+    - '#DiaSource.ssObjectId'
+    referencedColumns:
+    - '#SSObject.ssObjectId'
+  - name: FK_DiaSource_Object
+    description: Link a DiaSource to its associated Object.
+    '@type': ForeignKey
+    columns:
+    - '#DiaSource.diaObjectId'
+    referencedColumns:
+    - '#DiaObject.diaObjectId'
```

Comment:
Remove; duplicates FK_DiaSource_DiaObject, and there is currently no DiaSource-Object relationship